src/styles/sidebar/components/annotation-quote.scss

```diff
@@ -9,13 +9,9 @@
 
   &__quote {
     @include var.font-normal;
-
     border-left: 3px solid var.$grey-3;
     color: var.$color-text-light;
-    font-family: sans-serif;
-    font-size: 12px;
     font-style: italic;
-    letter-spacing: 0.1px;
```

Comment:
Typographer Frederic Goudy infamously lamented that “Anyone who would letterspace lowercase would steal sheep”.